collections/_article/dev-snapshot-godot-4-5-beta-2.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Dev snapshot: Godot 4.5 beta 2"
+excerpt: The cow goes "moo"! The duck goes "quack"! The bug goes *squash*!
+categories: [pre-release]
+author: Thaddeus Crews
+image: /storage/blog/covers/dev-snapshot-godot-4-5-beta-2.webp
+image_caption_title: Xion Leak
+image_caption_description: A game by Gentle Beasts Studio
+date: 2025-07-01 12:00:00
+---
+
+It's been two weeks since the release of [4.5 beta 1](/article/dev-snapshot-godot-4-4-beta-1/), and already we've been seeing a plethora of the various bugs and regressions dealt with. This is an ongoing process of course, so there's going to be a couple more beta snapshots following this, but progress has been smooth and we're quite satisfied with our current timeframe; great work, everyone!
```

Comment:
```suggestion
It's been two weeks since the release of [4.5 beta 1](/article/dev-snapshot-godot-4-4-beta-1/), and already a plethora of various bugs and regressions have been dealt with, so it's time for 4.5 beta 2. This is an ongoing process of course, so there's going to be a couple more beta snapshots following this, but progress has been smooth and we're quite satisfied with our current timeframe; great work, everyone!
```